az-cli/src/aro-preview/aro_preview.egg-info/PKG-INFO

```diff
@@ -0,0 +1,35 @@
+Metadata-Version: 1.1
```

Comment:
I think, `egg-info` can be ignored

Reply:
Yes, this whole directory should be removed from source control and gitignore'd.